specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/PrivateEndpointConnectionUpdate.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default",
+    "serverName": "test-svr",
+    "privateEndpointConnectionName": "private-endpoint-connection-name",
+    "api-version": "2018-06-01-preview",
+    "parameters": {
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description":  "Approved by johndoe@company.com"
```

Comment:
You know company.com is real? I don't think they want us to use them as an example ;) Please use known examples such as contoso or example.com. "admin@example.com" would be perfect.

Reply:
removed from this PR. Will address when I create PLR PR